owlbot.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import synthtool as s
+import synthtool.gcp as gcp
+from synthtool.languages import python
+
+# ----------------------------------------------------------------------------
+# Copy the generated client from the owl-bot staging directory
+# ----------------------------------------------------------------------------
+
+default_version = "v1"
+
+for library in s.get_staging_dirs(default_version):
+    s.move(library, excludes=["setup.py", "README.rst"])
+s.remove_staging_dirs()
+
+# Work around gapic generator bug https://github.com/googleapis/gapic-generator-python/issues/1083
+s.replace(
+    "google/cloud/**/types/compute.py",
+    """A request message for InstanceGroupManagers.AbandonInstances.
+    See the method description for details.\n
+    Attributes""",
+    """A request message for InstanceGroupManagers.AbandonInstances.
+    See the method description for details.\n
+    .. _oneof: https://proto-plus-python.readthedocs.io/en/stable/fields.html#oneofs-mutually-exclusive-fields\n
+    Attributes"""
+)
+
+# Work around formatting issues with docstrings
+s.replace("google/cloud/**/types/compute.py", """\"IT_\"""", """`IT_`""")
+s.replace("google/cloud/**/types/compute.py", """\"NS_\"""", """`NS_`""")
+
+# ----------------------------------------------------------------------------
+# Add templated files
+# ----------------------------------------------------------------------------
+
+templated_files = gcp.CommonTemplates().py_library(
+    microgenerator=True,
+    cov_level=98,
```

Comment:
Is the generated code missing coverage?

Reply:
Yes, I've filed https://github.com/googleapis/python-compute/issues/151. The current coverage is 96% so this PR brings coverage up by 2%. See https://source.cloud.google.com/results/invocations/a1055f0e-163e-49a7-8404-f0c81b7f53a3/targets